src/messages/ru/hipanel:finance.php

```diff
@@ -23,6 +23,11 @@
     'Servers' => 'Сервера',
     'Charges' => 'Списания',
     'Toggle sign' => 'Переключить знак',
+    'Opening balance' => 'Баланс на начало периода',
+    'Closed balance' => 'Суммарный балланс',
```

Comment:
https://osome.com/uk/term/closing-balance-uk/ (or google for closing balance and translations)

```suggestion
    'Opening balance' => 'Начальный баланс',
    'Closing balance' => 'Конечный балланс',
```